### PR TITLE
Update links to 2022.2

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: demos/speech_recognition_with_kaldi_model/python/README.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ pip3 install -r requirements.txt
 ### Getting ready with rm_lstm4f stateful model
 
 To run this example you will need to download the rm_lstm4f model with input and score ark files and convert it to IR format.
-- Download the model from [rm_lstm4f](https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/)
+- Download the model from [rm_lstm4f](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/)
 
 ```bash
 mkdir models && cd models
-wget -r -np -nH --cut-dirs=5 -R *index.html* https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/ 
+wget -r -np -nH --cut-dirs=5 -R *index.html* (https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/
```
Comment (mzegla) [inline demos/speech_recognition_with_kaldi_model/python/README.md:29]: Shouldn't it be like this?
```suggestion
wget -r -np -nH --cut-dirs=5 -R *index.html* https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/ 
```

File: docs/stateful_models.md [hunk 2]
```diff
@@ -22,11 +22,11 @@ Some models might take the whole sequence of data as an input and iterate over t
 
 Serving stateful model in OpenVINO Model Server is very similar to serving stateless models. The only difference is that for stateful models you need to set `stateful` flag in the model configuration.
 
-* Download and prepare example model from [rm_lstm4f](https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/)
+* Download and prepare example model from [rm_lstm4f]((https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/)
 
 ```bash
 mkdir models && cd models
-wget -r -np -nH --cut-dirs=5 -R *index.html* https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/
+wget -r -np -nH --cut-dirs=5 -R *index.html* (https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/
```
Comment (mzegla) [inline docs/stateful_models.md:29]:
```suggestion
wget -r -np -nH --cut-dirs=5 -R *index.html* https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/
```

File: demos/speech_recognition_with_kaldi_model/python/README.md [hunk 3]
```diff
@@ -24,27 +24,25 @@ pip3 install -r requirements.txt
 To run this example you will need to download the rm_lstm4f model with input and score ark files and convert it to IR format.
 - Download the model from [rm_lstm4f](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/)
 
-```bash
-mkdir models && cd models
```
Comment (ngrozae) [inline demos/speech_recognition_with_kaldi_model/python/README.md:28]: "mkdir models && cd models" command is still needed

File: demos/speech_recognition_with_kaldi_model/python/README.md [hunk 4]
```diff
@@ -24,27 +24,25 @@ pip3 install -r requirements.txt
...
 
-```
-test_feat_1_10.ark
+```bash
+wget https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/test_feat_1_10.ark
 ```
 
 [Kaldi's](http://kaldi-asr.org/doc/io.html) binary archive file with input data for the model
 
 ```
```
Comment (ngrozae) [inline demos/speech_recognition_with_kaldi_model/python/README.md:44]: if this file needs to be downloaded this part should be marked with "bash"

File: demos/speech_recognition_with_kaldi_model/python/README.md [hunk 5]
```diff
@@ -22,34 +22,32 @@ pip3 install -r requirements.txt
 ### Getting ready with rm_lstm4f stateful model
 
 To run this example you will need to download the rm_lstm4f model with input and score ark files and convert it to IR format.
-- Download the model from [rm_lstm4f](https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/)
+- Download the model from [rm_lstm4f](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/)
 
-```bash
-mkdir models && cd models
```
Comment (ngrozae) [inline demos/speech_recognition_with_kaldi_model/python/README.md:28]: "mkdir models && cd models" command is still needed

**Review by dtrawins — APPROVED** [review]

File: docs/stateful_models.md [hunk 6]
```diff
@@ -22,11 +22,13 @@ Some models might take the whole sequence of data as an input and iterate over t
 
 Serving stateful model in OpenVINO Model Server is very similar to serving stateless models. The only difference is that for stateful models you need to set `stateful` flag in the model configuration.
 
-* Download and prepare example model from [rm_lstm4f](https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/)
+* Download and prepare example model from [rm_lstm4f]((https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/)
```
Comment (mzegla) [inline docs/stateful_models.md:25]:
```suggestion
* Download and prepare example model from [rm_lstm4f](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/)
```

**Review by mzegla — APPROVED** [review]

File: tests/functional/model/models_information.py [hunk 7]
```diff
@@ -109,8 +101,7 @@ class Resnet:
     output_shape = (1, 1001)
     rest_request_format = 'column_name'
     model_path = os.path.join(config.models_path, name)
-    url = URL_PUBLIC_MODEL_FORMAT.format(repo=MODEL_REPOSITORY_SERVER, opencv=OPENCV_PUBLIC, build=BUILD_012020,
-                                         model=RESNET_50, model_version=RESNET_V1_50)
+    url = "https://storage.openvinotoolkit.org/repositories/open_model_zoo/public/2022.1/resnet-50-tf/" + RESNET_V1_50
```
Comment (jasiu86) [inline tests/functional/model/models_information.py:104]: url = f"https://storage.openvinotoolkit.org/repositories/open_model_zoo/public/2022.1/resnet-50-tf/{RESNET_V1_50}"

Comment (jasiu86) [reply tests/functional/model/models_information.py:104]: /2022.1/ is correct?

File: docs/stateful_models.md [hunk 8]
```diff
@@ -22,11 +22,13 @@ Some models might take the whole sequence of data as an input and iterate over t
 
 Serving stateful model in OpenVINO Model Server is very similar to serving stateless models. The only difference is that for stateful models you need to set `stateful` flag in the model configuration.
 
-* Download and prepare example model from [rm_lstm4f](https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/)
+* Download and prepare example model from [rm_lstm4f](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/)
```
Comment (jasiu86) [inline docs/stateful_models.md:25]: /2021.2/  not /2022.2/

File: docs/ovms_quickstart.md [hunk 9]
```diff
@@ -1,10 +1,10 @@
 # Quickstart Guide {#ovms_docs_quick_start_guide}
 
-OpenVINO Model Server can perform inference using pre-trained models in either [OpenVINO IR](https://docs.openvino.ai/2022.1/openvino_docs_MO_DG_IR_and_opsets.html#doxid-openvino-docs-m-o-d-g-i-r-and-opsets) 
+OpenVINO Model Server can perform inference using pre-trained models in either [OpenVINO IR](https://docs.openvino.ai/2022.2/openvino_docs_MO_DG_IR_and_opsets.html#doxid-openvino-docs-m-o-d-g-i-r-and-opsets) 
 or [ONNX](https://onnx.ai/) format. You can get them by:
 
-- downloading proper models from [Open Model Zoo](https://download.01.org/opencv/2021/openvinotoolkit/2021.1/open_model_zoo/models_bin/)
-- converting other formats using [Model Optimizer](https://docs.openvino.ai/2022.1/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html)
+- downloading proper models from [Open Model Zoo](https://storage.openvinotoolkit.org/repositories/open_model_zoo/public/2022.1/)
```
Comment (jasiu86) [inline docs/ovms_quickstart.md:6]: 2022.1 ?

File: demos/speech_recognition_with_kaldi_model/python/README.md [hunk 10]
```diff
@@ -22,34 +22,33 @@ pip3 install -r requirements.txt
...
+wget https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/rm_lstm4f.nnet
+wget https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/rm_lstm4f.mapping
 ```
 
-rm_lstm4f model files in Kaldi format.
+rm_lstm4f model files in Kaldi format:
 
-```
-test_feat_1_10.ark
+```bash
```
Comment (jasiu86) [inline demos/speech_recognition_with_kaldi_model/python/README.md:39]: 2022.1 or 2021.1 ?

File: demos/speech_recognition_with_kaldi_model/python/README.md [hunk 11]
```diff
@@ -22,34 +22,33 @@ pip3 install -r requirements.txt
 ### Getting ready with rm_lstm4f stateful model
 
 To run this example you will need to download the rm_lstm4f model with input and score ark files and convert it to IR format.
-- Download the model from [rm_lstm4f](https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/)
+- Download the model from [rm_lstm4f](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/rm_lstm4f/)
```
Comment (jasiu86) [inline demos/speech_recognition_with_kaldi_model/python/README.md:25]: 2021.2 ?

File: demos/image_classification/python/README.md [hunk 12]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-The script [image_classification.py](https://github.com/openvinotoolkit/model_server/blob/releases/2022/1/demos/image_classification/python/image_classification.py) reads all images and their labels specified in the text file. It then classifies them with [ResNet50](https://docs.openvino.ai/2022.1/omz_models_model_resnet50_binary_0001.html) model and presents accuracy results.
+The script [image_classification.py](https://github.com/openvinotoolkit/model_server/blob/releases/2022/1/demos/image_classification/python/image_classification.py) reads all images and their labels specified in the text file. It then classifies them with [ResNet50](https://docs.openvino.ai/2022.2/omz_models_model_resnet50_binary_0001.html) model and presents accuracy results.
```
Comment (jasiu86) [inline demos/image_classification/python/README.md:5]: 2022/1/ ?

**Review by jasiu86 — CHANGES_REQUESTED** [review]
Please verify comments related to 
2021.1

File: docs/stateful_models.md (hunk 8 above)
Comment (dtrawins) [reply docs/stateful_models.md:25]: contrib model is not present in 2022.2 folder

**Review by jasiu86 — APPROVED** [review]

**Pull request merged**